app/controllers/kudos_controller.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+class KudosController < ApplicationController
+  before_action :set_kudos, only: %i[show edit update destroy]
+  before_action :authenticate_employee!, except: %i[index show]
+  before_action :correct_employee, only: %i[edit update destroy]
```

Comment:
Actually no-one is using callbacks (apart from authentication) - they are making code much harder to read. Let's leave it like this for now - when you will come back some time in the future to this controller, and try to do something with it you will see it by yourself ;)

Reply:
I will change it in the next tasks :)